pkg/goDB/bigendian/readint_asm.go

```diff
@@ -14,10 +14,14 @@
 
 package bigendian
 
+// ReadUint64At returns the uint64 stored at location idx in b
 func ReadUint64At(b []byte, idx int) uint64
 
+// ReadInt64At returns the int64 stored at location idx in b
 func ReadInt64At(b []byte, idx int) int64
 
+// UnsafeReadUint64At returns the int64 stored at location idx in b. It accesses the data directly via the unsafe package
```

Comment:
Just a sidenote: We might reconsider replacing these with Go standard library little-endian storage / readout - in some recent benchmarks I realized that with recent Go versions there doesn't seem to be any performance benefit anymore (actually, in some cases even a penalty)...

Reply:
Sweet! I propose a new issue to play around with implementations.